### DISCUSSION TITLE
Inserting an Image into the portfolio

### DISCUSSION THREAD

**Discussion by ellenalamont17** [earthlab-education/Earth-Analytics-2023-01-Intro#124] [Q&A01 Online Portfolio] [answered]
I am struggling to get an imaged loaded into my portfolio page from a file specifically. I was able to get an online hosted image to appear.  Does it have to hosted online to work? I tried the directory filepath in the code in place of the URL, but I keep getting a broken link. If it has to be hosted online, where is a good place to host an image for this course?

**Comment by edchanonline** [comment, marked as answer]
The suggestion from the lecture is to upload your pictures to your Github repository. Here's a good example at https://github.com/eggvoice/eggvoice.github.io/tree/main/img

**Reply by edchanonline** [reply]
The part about creating a folder directory within your repo is not obvious. You have to upload a placeholder file with a sub folder under your root directory
![Screenshot 2023-09-04 at 8 47 02 PM](https://github.com/earthlab-education/Earth-Analytics-2023-01-Intro/assets/4795392/e719e84b-d34f-4512-aa67-329bdb5540ba)
